src/azul/oauth2.py

```diff
@@ -0,0 +1,4 @@
+class Oauth2Client:
```

Comment:
This is anemic and should go. If sth like this is needed in a future PR, it should come in the form of a controller.